jenkins/Pipeline.gy

```diff
@@ -51,8 +52,8 @@ pipeline {
             steps {
                 sh 'buildah login -u="${PB_CI_REG_CRED_USR}" -p="${PB_CI_REG_CRED_PSW}" ${PB_CI_REGISTRY}'
                 sh 'bash -ex ./server/pbenchinacan/container-build-ci.sh'
-                sh 'buildah push localhost/pbench-server:${PB_IMAGE_TAG} ${PB_CI_REGISTRY}/${PB_ORG_NAME}/pbench-server:${PB_IMAGE_TAG}'
-                sh 'buildah push localhost/${PB_SERVER_IMAGE_NAME}:${PB_IMAGE_TAG} ${PB_CI_REGISTRY}/${PB_ORG_NAME}/${PB_SERVER_IMAGE_NAME}:${PB_IMAGE_TAG}'
+                sh 'buildah push ${PB_CONTAINER_REG}/pbench-server:${PB_IMAGE_TAG} ${PB_CONTAINER_REG}/pbench-server:${PB_IMAGE_TAG}'
+                sh 'buildah push ${PB_CONTAINER_REG}/${PB_SERVER_IMAGE_NAME}:${PB_IMAGE_TAG} ${PB_CONTAINER_REG}/${PB_SERVER_IMAGE_NAME}:${PB_IMAGE_TAG}'
```

Comment:
If the source and the destination are the same, I don't think there is any value in specifying them both.  Consider dropping the second arguments to these two `push` commands, to get the line length back under control.

Reply:
Fixed.